site/guide/state.md

```diff
@@ -85,6 +85,19 @@ fn from_request(req: &'a Request<'r>) -> request::Outcome<T, ()> {
 
 [`Request::guard()`]: https://api.rocket.rs/rocket/struct.Request.html#method.guard
 
+### Request-Local State
+
+In addition to application-wide state, each `Request` has its own state container
+that can be used by request guards and fairings. This container can be used to
+cache expensive computations and lookups, such as to a database or external
+server, that might be used by multiple request guards during routing. Fairings
```

Comment:
Not just guards, but anything with access to a `Request`. Think of a way to word this more generally.